akka-docs/src/main/paradox/distributed-data.md

```diff
@@ -658,88 +658,7 @@ Java
-# 1. A relative or absolute path to a directory that ends with 'ddata'
-#    the full name of the directory will contain name of the ActorSystem
-#    and its remote port.
-# 2. Otherwise the path is used as is, as a relative or absolute path to
-#    a directory.
-akka.cluster.distributed-data.durable.lmdb.dir = "ddata"
-```
-
-Java
-:   ```
-# Directory of LMDB file. There are two options:
-# 1. A relative or absolute path to a directory that ends with 'ddata'
-#    the full name of the directory will contain name of the ActorSystem
-#    and its remote port.
-# 2. Otherwise the path is used as is, as a relative or absolute path to
-#    a directory.
-akka.cluster.distributed-data.durable.lmdb.dir = "ddata"
-```
-
-
-When running in production you may want to configure the directory to a specific
-path (alt 2), since the default directory contains the remote port of the
-actor system to make the name unique. If using a dynamically assigned
-port (0) it will be different each time and the previously stored data
-will not be loaded.
-
-Making the data durable has a performance cost. By default, each update is flushed
-to disk before the `UpdateSuccess` reply is sent. For better performance, but with the risk of losing
-the last writes if the JVM crashes, you can enable write behind mode. Changes are then accumulated during
-a time period before it is written to LMDB and flushed to disk. Enabling write behind is especially
-efficient when performing many writes to the same key, because it is only the last value for each key
-that will be serialized and stored. The risk of losing writes if the JVM crashes is small since the
-data is typically replicated to other nodes immediately according to the given `WriteConsistency`.
-
-```
-akka.cluster.distributed-data.durable.lmdb.write-behind-interval = 200 ms
-```
-
-Note that you should be prepared to receive `WriteFailure` as reply to an `Update` of a
-durable entry if the data could not be stored for some reason. When enabling `write-behind-interval`
-such errors will only be logged and `UpdateSuccess` will still be the reply to the `Update`.
-
-There is one important caveat when it comes pruning of [CRDT Garbage](#crdt-garbage) for durable data.
-If an old data entry that was never pruned is injected and merged with existing data after
-that the pruning markers have been removed the value will not be correct. The time-to-live
-of the markers is defined by configuration
-`akka.cluster.distributed-data.durable.remove-pruning-marker-after` and is in the magnitude of days.
-This would be possible if a node with durable data didn't participate in the pruning
-(e.g. it was shutdown) and later started after this time. A node with durable data should not
-be stopped for longer time than this duration and if it is joining again after this
-duration its data should first be manually removed (from the lmdb directory).
+See @ref:[Durable Storage](typed/distributed-data.md#durable-storage).
```

Comment:
I think "See the typed [Durable Storage]" would be better because now it looks like this:

<img width="318" alt="Screen Shot 2019-09-20 at 10 17 05" src="https://user-images.githubusercontent.com/666915/65311037-d319cb00-db8f-11e9-90f7-fb0321ffb595.png">

For all of those cross-refs above and below as well.

Reply:
Agree that it looks funny, we have also said that we shouldn't use Typed in too many places.
Maybe: See [Durable Storage] in the documentation of the new APIs.

Reply:
Yes, it is one of those issues in moving from A to B pointers that we hit, how to best do it, without using "typed". The overarching ticket is to not use "Typed". Will go with `new`
```See [Durable Storage] in the documentation of the new APIs.```